scripts/00spinup.py

```diff
@@ -57,10 +62,10 @@
 # Initialization
 # ------------------------------------------
 # Arrays for total values (across all layers), with time as the first axis
-subsurface_storage = np.zeros((nt,))
-surface_storage = np.zeros((nt,))
+subsurface_storage = np.zeros(nt)
```

Comment:
I didn't realize we don't need a tuple-syntax for shape for a 1-d array - I agree, this syntax is simpler.